src/sentry/buffer/redis.py

```diff
@@ -293,7 +298,7 @@ def _process_single_incr(self, key):
                 if k.startswith("i+"):
                     incr_values[k[2:]] = int(v)
                 elif k.startswith("e+"):
-                    if v.startswith("["):
+                    if v.startswith(b"["):
                         extra_values[k[2:]] = self._load_value(json.loads(v))
```

Comment:
Should `v` be decoded here?

Reply:
no json.joads takes bytes

Reply:
Hmm, that's a good question though, does it then return a byte dict

Reply:
You decode a string a on 289 before passing it to `json.loads`

Reply:
Okay, I've switched this over to also decoding the value before `loads`. I think you're right here.